Hearthstone/app/src/main/java/com/example/hearthstone/ui/homepage/HomeFragment.kt

```diff
@@ -21,11 +22,22 @@ class HomeFragment : Fragment() {
     private lateinit var binding: FragmentHomeBinding
     private val viewModel: HomeViewModel by viewModels()
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        activity?.onBackPressedDispatcher?.addCallback(this, object : OnBackPressedCallback(true){
```

Comment:
Add logging here when a user wants to press back on base activity. I'm assuming this stops the application from closing?

Reply:
Yes, it was the method I found to prevent the app from closing when the user presses the "back" button on the cell phone